# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../paragui/swig_fixes.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./include/pgbutton.h.orig	2005-05-09 23:40:53.000000000 +0200
+++ ./include/pgbutton.h	2005-05-09 23:42:23.000000000 +0200
@@ -201,7 +201,7 @@
 
 	// 1.0.1 versions (surface)
-#ifndef SWIG
+#if !defined SWIGRUBY && !defined SWIGPYTHON
	bool SetIcon(SDL_Surface* icon_up, SDL_Surface* icon_down = NULL);
 #endif
 
 	SDL_Surface* GetIcon(Uint8 num);	
